guardian/newsletters-nx
# Remove ophan status as nothing async happens

## What does this change?

This removes the async status for Ophan as there is no async process for Ophan so this is just noise. 
The property is left in the model as an optional so we do not get validation errors on the existing newsletters that have this property. We can, at a later date, run a backfill to remove this property from all the existing live newsletters and then remove the property altogether

This also  removed the Ophan element of the launch Wizard. Editorial launch newsletters and don't cre about this. We can edit the Campain code later if required. 

## How to test

Make sure we can create and launch newsletters and that we can view existing newsletters. (Draft and Live), and that the API response is as expected


## How can we measure success?

It still working

## Have we considered potential risks?

Should be fine - nothing externally cares about these properties

## Images

![Screenshot 2024-06-10 at 11 30 10](https://github.com/guardian/newsletters-nx/assets/3277259/60ba8a95-1024-46ca-a70b-6350a0f00493)
